openweathermap-ios/openweathermap-ios/MapViewController.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  MapViewController.swift
+//  openweathermap-ios
+//
+
+
+import UIKit
+import MapKit
+
+final class MapViewController: UIViewController, CLLocationManagerDelegate {
+
+    @IBOutlet private weak var mapView: MKMapView!
+    private let locationManager = CLLocationManager()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        mapView.delegate = self
+        locationManager.delegate = self
+        locationManager.desiredAccuracy = kCLLocationAccuracyBest
+        locationManager.requestWhenInUseAuthorization()
+        locationManager.startUpdatingLocation()
+        
+    }
+    
+    
+    func addAnnotation(location: CLLocationCoordinate2D){
```

Comment:
Prefer to use _updateAnnotation_ instead of _addAnnotation_ because the _add_ suffix passes the idea of increment.

Reply:
This isn't resolved.

Reply:
I marked it as resolved because I resolved it in my code, I just haven't pushed it yet. I marked it as resolved to have control :).
I will mark only as resolved after the PR.

Reply:
This is a bad practice. Consider pushing to keep the control. The PR is public. Your machine doesn't. If you resolve without pushing, anyone besides me can see the same issue and repeat the comment unnecessarily.